cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/CgmesModel.java

```diff
@@ -29,6 +29,8 @@
 
     Properties getProperties();
 
+    PropertyBags fullModel(String cgmesProfile);
```

Comment:
Add a default implementation to prevent breaking changes

Reply:
Done